Resources/Prototypes/_Mono/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/projectiles.yml

```
@@ -1,10 +1,11 @@
 - type: entity
   id: ShipMediumPlasmaProjectile
   name: plasma projectile
-  parent: BaseBullet
+  parent: BaseBulletTrigger
   categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)